src/corporacreator/preprocessors/cy.py

```diff
@@ -1,12 +1,38 @@
-def cy(client_id, sentence):
-    """Cleans up the passed sentence, removing or reformatting invalid data.
+# compile the set of the alphabet just once
+alphabet = set([' ', 'a','á', 'à', 'â', 'ä', 'b', 'c',
+                'd', 'e', 'é', 'è', 'ê', 'ë', 'f', 'g',
+                'h', 'i', 'í', 'ì', 'î', 'ï', 'j', 'l',
+                'm', 'n', 'o', 'ó', 'ò', 'ô', 'ö', 'p',
+                'r', 's', 't', 'u', 'ú', 'ù', 'û', 'ü',
+                'w', 'ẃ', 'ẁ', 'ŵ', 'ẅ', 'y', 'ý', 'ỳ',
+                'ŷ', 'ÿ'])
 
+def cy(client_id, sentence):
+    """
+    Cleans up the passed sentence, removing or reformatting invalid data.
     Args:
       client_id (str): Client ID of sentence's speaker
       sentence (str): Sentence to be cleaned up.
+      alphabet (set): set of unique, lowercased characters in language.
```

Comment:
This should only document the arguments to the function `cy()`, not the global variable `alphabet`.

Reply:
resolved in latest commit